docs/release-process.rst

```diff
@@ -1,14 +1,30 @@
 Magic-Folder Release Process
 ============================
 
-This document is an initial effort towards providing documentation for a magic-folder release process.
-See `https://github.com/LeastAuthority/magic-folder/issues/532`_ for details about improvements.
+There is no schedule for magic-folders releases.
+We endeavour to keep ``main`` as always releasable.
 
-The release process involves at least the following steps:
 
-* Update the package version number.
-  This must be done in at least:
+Versioning Scheme
+-----------------
 
-  * the ``setup`` call in ``setup.py``
-  * the ``buildPythonPackage`` call in ``nix/default.nix``
-  * the ``__version__`` definition in ``src/magic_folder/__init__.py``
+We use a kind of Calendar Versioning (`https://calver.org/`_):
```

Comment:
My personal hesitation with using calver over semver is that it doesn't express breaking or backward incompatible changes (and, with a project like Magic-Folder that hasn't yet reached full maturity, we should probably expect such changes at some point in the future). Does Magic-Folder have any sort of policy about breaking changes, API stability guarantees, etc. -- and, specifically, about how to communicate those things via the release process? Maybe it's still too early to think about those things now, but consider including any thoughts about these sorts of things in this doc if you have them.

Reply:
okay, i've made a "stability" statement -- does it make sense?

Reply:
Sounds great. :+1: